### PR TITLE
Updated schema.org information

### PR DESCRIPTION
**Pull request by clyons**
_config.yml:

* Added telephone and address variables

resume.html:

* Removed invalid property ‘contact-info’
* Changed person to ‘alumniOf’ CollegeOrUniversity, instead of ‘worksFor’
* Changed itemtype of ‘CreativeWork’ to Projects
* Added ‘creator’ property to Projects
* Added itemtype of ‘Organization’ to Associations
* Added ‘memberOf’ property to Associations
* Updated social links schema to ‘sameAs’ instead of ‘url’ ([ref](https://developers.google.com/search/docs/data-types/social-profile-links))

Structured Data Testing Tool Results:

screenshot 2017-08-10 14 57 01
screenshot 2017-08-10 14 57 19

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -12,8 +12,9 @@ sass:
 resume_avatar:                  "true"
 resume_name:                    "Homer J. Simpson"
 resume_title:                   "Nuclear Safety Inspector"
-resume_contact_email:
-  "homerjsimpson@youremailaddress.com"
+resume_contact_email:           "homerjsimpson@youremailaddress.com"
+resume_contact_telephone:       "555-7334"
+resume_contact_address:         "742 Evergreen Terrace"
```
Comment (jglovier) [inline _config.yml]: Any reason this is just the street address?

File: _includes/icon-links.html [hunk 2]
```diff
@@ -4,32 +4,32 @@
 
   <!-- GitHub link -->
   {% if site.resume_social_links.resume_github_url %}
-  <li class="icon-link-item"><a href="{{ site.resume_social_links.resume_github_url }}" class="icon-link" itemprop="url">{% include icons/icon-github.html %}</a></li>
+  <li class="icon-link-item"><a href="{{ site.resume_social_links.resume_github_url }}" class="icon-link" itemprop="sameAs">{% include icons/icon-github.html %}</a></li>
```
Comment (jglovier) [inline _includes/icon-links.html:7]: Not sure I understand the difference between the `url` value and the `sameAs` value. I looked it up on schema.org, but it's not very clear there lol. Can you help me understand?

**Review by jglovier — COMMENTED** [review]
Looks good! Thanks for adding these.

Two things:
- Can you please resolve the conflicts
- Can you please post a screenshot showing how the rendered view changes with this PR

Thanks! ✨

File: _config.yml (hunk 1 above)
Comment (clyons) [reply _config.yml]: Ah - used the address information from resume_header_contact_info, but I see I left out the "Springfield" part. Sorry for the confusion.

**Comment by clyons** [comment]
No problem. The best information on that is here:

https://developers.google.com/search/docs/data-types/social-profile-links

> In addition to the general structured data guidelines, Social Profile markup should follow these guidelines:
> 
> Markup must include a Person or Organization record with the following properties:
> * url—the url of your official website
> * sameAs—the urls of your official social media profile pages
>
> The social profiles in your markup must correspond to what users see on the same page.

All going well, this markup will add your social profile information to the Google Knowledge panel, like so:

File: _includes/icon-links.html (hunk 2 above)
Comment (clyons) [reply _includes/icon-links.html:7]: No problem. The best information on that is here:

https://developers.google.com/search/docs/data-types/social-profile-links

> In addition to the general structured data guidelines, Social Profile markup should follow these guidelines:
> 
> Markup must include a Person or Organization record with the following properties:
> * url—the url of your official website
> * sameAs—the urls of your official social media profile pages
>
> The social profiles in your markup must correspond to what users see on the same page.

All going well, this markup will add your social profile information to the Google Knowledge panel, like so:

**Comment by clyons** [comment]
Conflicts resolved.

Screenshot follows. There should be zero visible changes with this commit - everything is designed to be invisible and backwards-compatible with the previous version.

screencapture-127-0-0-1-4000-1502407985058

File: _includes/icon-links.html (hunk 2 above)
Comment (jglovier) [reply _includes/icon-links.html:7]: Ahhhh gotcha! That's great. Thanks for updating this, and teaching me something new! :sparkles:

**Comment by jglovier** [comment]
Looks great. Thanks again, @clyons!! :shipit:

**Pull request merged**